components/picker/stories/picker.stories.js

```diff
@@ -26,23 +29,55 @@ export default {
 			type: { name: "string" },
 			table: {
 				type: { summary: "string" },
-				category: "Content",
+				category: "Component",
 			},
-			options: ["top", "left"],
+			options: ["top", "side"],
 			control: { type: "select" },
 		},
 		withSwitch: {
-			name: "Display with a switch component",
+			name: "Show switch component",
+			description: "Display a separate switch component after the picker. Helpful for testing alignment with the picker when using the side label.",
 			type: { name: "boolean" },
 			table: {
 				type: { summary: "boolean" },
-				category: "Component",
+				category: "Advanced",
+			},
+			control: "boolean",
+			if: { arg: "labelPosition", eq: "side" },
+		},
+		showWorkflowIcon: {
+			name: "Show workflow icon",
+			description: "Display optional workflow icon before the value or placeholder",
+			type: { name: "boolean" },
+			table: {
+				type: { summary: "boolean" },
+				category: "Advanced",
 			},
 			control: "boolean",
-			if: { arg: "labelPosition", eq: "left" },
 		},
 		placeholder: {
 			name: "Placeholder",
+			description: "The placeholder text prompts a user to select an option from the picker menu. It disappears once a user selects an option. This will not be displayed if the `value` control is set.",
+			type: { name: "string" },
+			table: {
+				type: { summary: "string" },
+				category: "Content",
+			},
+			control: { type: "text" },
+		},
+		currentValue: {
+			name: "Value",
+			description: "The value shows the option that a user has selected.",
+			type: { name: "string" },
+			table: {
+				type: { summary: "string" },
+				category: "Content",
+			},
+			control: { type: "text" },
+		},
+		helpText: {
+			name: "Help text",
+			description: "Optional help text that can be informational or an error message. For error messages, also set the invalid control to true. Displays a separate help text component after the picker.",
```

Comment:
An opinion: I read this slightly oddly at first, and I wonder if we could re-word it? The verb in the 2nd sentence is "set" and the verb in the 3rd sentence is "displays", which don't agree and that threw me off. Using "sets" instead of "set" changes the meaning, so that's not a good change. Maybe we could flip the 2nd and 3rd sentence? Like:

```suggestion
			description: "Optional help text that can be informational or an error message. Displays a separate help text component after the picker. For error messages, also set the invalid control to true.",
```

and/or rearrange the words a bit?

```suggestion
			description: "Optional help text that can be informational or an error message. Displays a separate help text component after the picker. For error messages, also the invalid control must be set to true.",
```